src/building/how-to-build-and-run.md

```diff
@@ -98,6 +164,24 @@ build. The **full** `rustc` build (what you get with `./x.py build
 
 You almost never need to do this.
 
+## Incremental builds
+
+You can configure rustbuild to use incremental compilation with the
+`--incremental` flag:
+
+```sh
+$ ./x.py build --incremental
+```
+
+The `--incremental` flag will store incremental compilation artifacts
+in `build/<host>/stage0-incremental`. Note that we only use incremental
+compilation for the stage0 -> stage1 compilation -- this is because
+the stage1 compiler is changing, and we don't try to cache and reuse
+incremental artifacts across different versions of the compiler.
+
+You can always drop the `--incremental` to build as normal (but you
+will still be using the local nightly as your bootstrap).
```

Comment:
This parenthetical about nightly seems wrong to me (bootstrapping is usually done with the downloaded beta), unless I'm misunderstanding its intent.

Reply:
Fixed